timeline/transform.py

```diff
@@ -40,16 +40,10 @@ def merge_start_end_events(timeline):
                 'sexp_idx': start_event['sexp_idx'],
                 'start_time': start_event['time'],
                 'end_time': end_event['time'],
-                'total_time': {
-                    'secs': end_event['time']['secs'] - start_event['time']['secs'],
-                    'nanos': end_event['time']['nanos'] - start_event['time']['nanos']
-                }
+                'total_ms': int(1e3 * (end_event['time']['secs'] - start_event['time']['secs']) +
```

Comment:
Can we just have it store the time in ms directly from the rust and then we wouldn't need to do this computation ourselves here?

Reply:
take a look at https://docs.rs/serde_with/latest/serde_with/struct.DurationMilliSeconds.html
(and https://docs.rs/serde_with/latest/serde_with/struct.DurationSeconds.html for more explanation/examples)